pom.xml

```diff
@@ -478,6 +652,29 @@
               <artifactId>maven-project-info-reports-plugin</artifactId>
               <version>3.0.0</version>
             </plugin>
+
+            <plugin>
+              <groupId>org.jacoco</groupId>
```

Comment:
Why does the Javadoc profile have a Jacoco configuration?

Reply:
Good catch. This was before I split jacoco and javadoc into separate profiles.